app/ashlar/ashlar/models.py

```diff
@@ -0,0 +1,60 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import ForeignKey, Column, create_engine
+from sqlalchemy.types import Integer, String, Float, DateTime
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy import func, UniqueConstraint
+from sqlalchemy.schema import Index
+
+import jsonschema
+
+from geoalchemy2.types import Geometry
+
+
+BaseModel = declarative_base()
+
+
+class AshlarModel(BaseModel):
+    __abstract__ = True
+
+    created = Column(DateTime(timezone=True), nullable=False, default=func.now())
+    modified = Column(DateTime(timezone=True), nullable=False, server_onupdate=func.now(), default=func.now())
+    occurred_from = Column(DateTime(timezone=True), nullable=False, default=func.now())
+    occurred_to = Column(DateTime(timezone=True), nullable=False, default=func.now())
```

Comment:
Yeah, sorry, I put that comment in an unclear spot. I did mean just the `occurred_*` fields.